src/client/eppo-client-assignment-details.spec.ts

```diff
@@ -77,22 +77,24 @@ describe('EppoClient get*AssignmentDetails', () => {
       },
       matchedAllocation: {
         key: 'targeted allocation',
+        name: 'Allocation for targeted allocation',
         allocationEvaluationCode: AllocationEvaluationCode.MATCH,
-        orderPosition: 0,
+        orderPosition: 1,
```

Comment:
The "orderPosition" changes were tests that needed to be updated from a previous change